v2/session/common-customizations/core/add-ssl-via-nginx.mdx

```diff
@@ -102,13 +97,13 @@ server {
         proxy_cache_bypass $http_upgrade;
   }
 }
-```
+  ```
 :::important
 If you decide to use self signed certificates, browsers will show a warning that the “Certificate is not Trusted”. Make sure to use a valid certificate in production.
```

Comment:
Remove this tool tip.. no mention of browser or frontend or anything of that sort here.